addons/rating/static/src/scss/rating_templates.scss

```diff
@@ -29,3 +29,9 @@
         }
     }
 }
+
+.div-center-y {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+}
```

Comment:
It is not completely removed. 🧐

Reply:
hello, i remember we showed the padding solution to Alexandra in the office and when zooming in, it wasn't really a good solution to center the text so if i'm not mistaken ( i can't find any trace as it was discussed offline ) we preferred to keep the actual solution. correct me if i'm wrong

Reply:
See in the chatter of the task.